notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p><em><u>test</u></em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p><strong>reply</strong></p>

Reply:
<ul><li>Descendant means anywhere nested within it in the DOM tree. Could be a direct child, could be five levels deep, it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level down.Descendant means anywhere nested within it in the DOM tree.</li><li>Could be a direct child, could be five levels deep, it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level down.Descendant means anywhere nested within it in the DOM tree. Could be a direct child, could be five levels deep,</li><li class="ql-indent-1">it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level down.Descendant means anywhere nested within it in the DOM tree. Could be a direct child, could be five levels deep, it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level down.Descendant means anywhere nested within it in the DOM tree.</li><li class="ql-indent-1">Could be a direct child, could be five levels deep, it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level</li><li class="ql-indent-1">down</li><li class="ql-indent-1">Descendant means</li><li class="ql-indent-2">anywhere</li><li class="ql-indent-2">nested within it in the DOM tree. Could be a direct child, could be five levels deep, it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level</li><li class="ql-indent-2">down</li><li class="ql-indent-2">Descendant means anywhere nested within it in the DOM tree. Could be a direct child, could be five levels deep, it is still a descendant. This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the</li><li class="ql-indent-2">next nested level down.Descendant means anywhere nested within it in the DOM tree. Could be a direct child, could be five levels deep, it is still a descendant.</li><li class="ql-indent-3">This is different than a&nbsp;<a href="https://css-tricks.com/almanac/selectors/c/child/" target="_blank">child combinator</a>&nbsp;(&gt;) which requires the element to be the next nested level</li><li class="ql-indent-3">down.</li></ul><p> </p>